.github/workflows/unit-tests.yml

```diff
@@ -9,8 +9,12 @@ on:
       - dev
     paths-ignore:
       - 'hail_search/**'
+      - '.github/workflows/hail-search-unit-tests.yaml'
   pull_request:
     types: [opened, synchronize, reopened]
+    paths-ignore:
+      - 'hail_search/**'
+      - '.github/workflows/hail-search-unit-tests.yaml'
```

Comment:
I thought this wasn't working correctly because the seqr unit tests are getting triggered, but this file is `unit-tests.yml`, which should trigger the tests.  🤦

Reply:
been there, done that, got the T shirt 😂